extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -493,7 +493,7 @@ export class AmpSlideScroll extends BaseSlides {
    */
   showSlideWhenReady_(value) {
     const index = parseInt(value, 10);
-    if (isFinite(index)) {
+    if (isFinite(index) && index > 0 && index < this.noOfSlides_) {
```

Comment:
Can't the `index` be `0`?

Further: is this error comes from predefined use case, such as swiping or auto-advance/loop? Or from user-defined user action (`on` action). If on-action - this should probably be `user().error()` if from predefined behavior - it should probably be a `dev.error`.